samples/display/ili9340/boards/stm32f429i_disc1.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_SPI_STM32=y
```

Comment:
This one is not needed, since `SPI_STM32` is set to y if `SPI` is set to y (done in prj.cnf)

Reply:
Thank you for noticing. I've removed the whole file as it is not required